Xamarin.Forms.Platform.Tizen/Renderers/CarouselViewRenderer.cs

```diff
@@ -0,0 +1,404 @@
+using ElmSharp;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+
+namespace Xamarin.Forms.Platform.Tizen
+{
+	class CarouselViewRenderer : ViewRenderer<CarouselView, Box>
+	{
+		INotifyCollectionChanged _observableSource = null;
+		List<View> _createdView = new List<View>();
+		List<IndexItem> _indicatorItems = new List<IndexItem>();
+
+		Native.Canvas _innerLayout;
+		Native.Scroller _scroller;
+		Index _indicator;
+
+		int _changedByScroll = 0;
+		int _isLayouting = 0;
+
+		int prevPoint = 0;
```

Comment:
maybe missing "_" prefix for local variable

Reply:
Fixed Thanks